tests/Resources/config/packages/db_tools_deprecated_v2_conflict.yaml

```diff
@@ -0,0 +1,9 @@
+#
+# Deprecated but yet supported configuration options in ^2.0.
```

Comment:
Wrong comment, this tests that when there is a conflict between old and new name, the configuration raises an exception.